Makefile

```
@@ -3,6 +3,7 @@
 test:
 	php -d "memory_limit=-1" vendor/bin/php-cs-fixer fix --ansi -vvv
 	php -d memory_limit=-1 vendor/bin/phpstan analyse -c phpstan.neon;
+	php vendor/bin/phpunit tests/
```

```suggestion
test:
	php -d "memory_limit=-1" vendor/bin/php-cs-fixer fix --ansi -vvv
	php -d "memory_limit=-1" vendor/bin/phpstan analyse -c phpstan.neon;
	php -d "memory_limit=-1" vendor/bin/phpunit tests/
```

I think your tests may need all the memory